cogs/welcome/welcome.py

```diff
@@ -537,6 +648,112 @@ def check(message: discord.Message):
         await self.config.guild(ctx.guild).get_attr(key).set(greetings)
         return
 
+    # [p]welcomeset greetings image
+    @greetings.group(name="image")
+    async def image(self, ctx: Context):
+        """Base command for the image command group"""
+        pass
+
+    # [p]welcomeset greetings image template
+    @image.command(name="template")
+    async def imageTemplate(self, ctx: Context):
+        await ctx.send(
+            "Here is the welcome image template so you can make your own! For best results please render the image at [IMAGE_DPI]dpi, [IMAGE_DIMENSION_HEIGHT] x [IMAGE_DIMENSION_WIDTH]. The bot will try to make it conform automatically but mileage may vary.",
+            file=discord.File(
+                os.path.join(
+                    data_manager.bundled_data_path(self),
+                    KEY_WELCOME_IMG_TEMPLATE,
+                )
+            ),
+        )
+
+    # [p]welcomeset greetings image add
+    @image.command(name="add")
+    async def imgAdd(self, ctx: Context, name: str):
+        """adds the attached image to the pool of random based images used to generate custom welcome images. Attaches only the first image attached.
+
+
+        Additionally automatically makes the sent image conform to the dimensions and dpi that's been tested for: 72dpi, 1193x671. Mileage may vary
+
+        """
+        await self.ensureCurrentServerHasImgCache(ctx.channel)
+        file_name = f"{name}.png"
+        fp = os.path.join(self.imgDir, str(ctx.guild.id), file_name)
```

Comment:
If `self.imgDir` is a `pathlib.Path` instance, we can make use of it instead of `os.path` functions.